anyway/widgets/suburban_widgets/accident_severity_by_cross_location_widget.py

```diff
@@ -43,3 +43,10 @@ def injury_severity_by_cross_location_mock_data():  # Temporary for Frontend
                 "killed_injury_severity_count": 0,
             },
         ]
+
+    @staticmethod
+    def localize_items(request_params: RequestParams, items: Dict) -> Dict:
+        items["data"]["text"] = {
+            "title": "Number of pedestrian accidents on Ben Yehuda street in Tel Aviv by crossing location"

```

Comment:
No translation?

Reply:
same, i mean what's the purpose of translating a very specific title (ben yehuda)?